src/Manager/Environment/environmentEffects.js

```diff
@@ -132,6 +144,21 @@ const download = (version, { name, sha512, uri }) => async dispatch =>
                                 new Error(`Checksum verification failed ${url}`)
                             );
                         }
+                        sendUsageData(
+                            EventAction.DOWNLOAD_TOOLCHAIN_SUCCESS,
+                            url
+                        );
+                        sendUsageData(
+                            EventAction.DOWNLOAD_TOOLCHAIN_TIME,
+                            `${calculateTimeConsumed(
+                                downloadTimeStart
+                            )} min; ${url}`
+                        );
+                        logger.info(
+                            `Finish downlaoding toolchain ${version} in approximate ${calculateTimeConsumed(
```

Comment:
[minor] Typo - "downlaoding"

Reply:
Also, the entire structure of the sentence should probably be changed to something like "Version ${version} of the toolchain will finish downloading in approximately ${timeConsumed} minutes".

Or something like that.

Reply:
Good catch and good suggestion, will update.